src/main/java/dan200/computercraft/core/lua/LuaJLuaMachine.java

```diff
@@ -412,6 +415,7 @@ public void execute()
                                             }
                                             catch( Throwable t )
                                             {
+                                                ComputerCraft.log.error( "Error running task", t);
```

Comment:
I was thinking of doing that, but I'd honestly prefer it was on - these bugs appear in the wild too, and having logging enabled at all times makes it easier to process user bug reports too. I'm not convinced that flooding a log via peripherals is going to be the most efficient way to bring down a server. That being said, I obviously don't want to provide tools which can be abused.